src/goTest.ts

```diff
@@ -236,3 +240,14 @@ function getTestFlags(goConfig: vscode.WorkspaceConfiguration, args: any): strin
 	let testFlags = goConfig['testFlags'] ? goConfig['testFlags'] : goConfig['buildFlags'];
 	return (args && args.hasOwnProperty('flags') && Array.isArray(args['flags'])) ? args['flags'] : testFlags;
 }
+
+function expandFilePathInOutput(output: string, cwd: string): string {
+	let lines = output.split('\n');
+	for (let i = 0; i < lines.length; i++) {
+		let matches = lines[i].match(/^\t(\w+_test.go):(\d+):/);
```

Comment:
You might not want to use `\w` here because it might not match all characters that are valid in a path, like `.`. Maybe `\S+` (non whitespace char) or just `.+` because it's searching between \t and `_test` already?